expense-backend/dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use the official Node.js image as the base image
+FROM node:20
+
+# Set the working directory in the container
+WORKDIR /backend
+
+# Copy package.json and package-lock.json to the container
+COPY package*.json tsconfig.json ./
+
+# Install dependencies
+RUN npm install
+
+# Copy the rest of the application code to the container
+COPY . .
+
+
+# Expose the port that the app runs on
+EXPOSE 3000
+
+# Command to run the application
+RUN NODE_ENV=development npm i
```

Comment:
Review again these steps.

1. First you wanna install
2. Then you wanna build
3. Finally you want to keep only important resources.